### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by Richard-Scap**
want to :ship:!! :hammer: :watch:

### CODE REVIEW

File: app/views/layouts/_nav.html.erb [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+		  </button>
+    	<%= link_to "Cerberus", root_path, class: "navbar-brand" %>
+    </div>
+
+    <div class="collapse navbar-collapse" id="navbar-collapse-1">
+    	<ul class="nav navbar-nav"> 
+    		<li class="dropdown">
+    			<a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Cities<span class="caret"></span></a>
+    			<ul class="dropdown-menu">
+    				<% @city.map do |n| %>
```
Comment (txssseal) [inline app/views/layouts/_nav.html.erb]: Instead of `map` use `each`

File: app/views/printers/index.html.erb [hunk 2]
```diff
@@ -1,7 +1,6 @@
-
-<p id="notice"><%= notice %></p>
-
-<h1>Listing Printers</h1>
+<%#binding.pry%>
```
Comment (txssseal) [inline app/views/printers/index.html.erb]: :Hank ey:

File: app/views/layouts/_nav.html.erb [hunk 3]
```diff
@@ -0,0 +1,50 @@
+<% provide(@city = City.all)  %>
```
Comment (Richard-Scap) [inline app/views/layouts/_nav.html.erb]: need to remove..

File: app/views/printers/index.html.erb (hunk 2 above)
Comment (txssseal) [inline app/views/printers/index.html.erb]: :hankey:

**Pull request merged**